lib/ansible/modules/identity/ipa/ipa_user.py

```diff
@@ -190,6 +195,15 @@ def get_user_diff(client, ipa_user, module_user):
     :param module_user:
     :return:
     """
+    non_updateable_keys = ['force']
```

Comment:
nit/non-blocking: Have you thought about adding `force` to the docstring?

Reply:
It is in the docstring on lines 26-30

Reply:
@mattymo @fxfitz is talking about Function documentation on line 195